app/models/booking.rb

```diff
@@ -8,12 +8,7 @@ class Booking < ApplicationRecord
   validates :garden_id, presence: true
   validates :status, inclusion: { in: STATUS }
 
-
```

Comment:
All of this is working and neede for the functioning of the validations and flatpickr

Reply:
that was changed